onnxruntime/core/providers/js/js_kernel.h

```diff
@@ -67,6 +67,7 @@ namespace js {
                             float value;                                           \
                             ORT_ENFORCE(info.GetAttr<float>(#attr_name, &value));, \
                                                                                  , ({#attr_name : $1}), static_cast<double>(value))
+#define JSEP_HEAP_PTR(ptr) reinterpret_cast<uintptr_t>(ptr)
```

Comment:
Would gsl::narrow be better? It can check the range and throw an exception when needed. Silently discarding data is dangerous. Especially in the setting that wasm programs are hard to debug.

Reply:
no range check is needed, I want the 'prt' (aka offset) as is. reinterpret_cast was what we used so I kept it but I think we don't need it, will test it.